MAUI/TreeMap/brush-settings.md

```diff
@@ -0,0 +1,438 @@
+---
+layout: post
+title: Brush settings in .NET MAUI TreeMap (SfTreeMap) control | Syncfusion
```

Comment:
title: Brush Settings in .NET MAUI TreeMap (SfTreeMap) Control | Syncfusion

Reply:
Updated.